public/content/lessons/2017/derivative-formulas-geometrically/index.mdx

```diff
@@ -1,10 +1,381 @@
+
+<Figure
+  image="figures/10.58.svg" 
+/>
+
+This is a nice way to think about the graph of $1/x$, by the way. If you think of the width $x$ of this puddle in the $xy$-plane, the corresponding output $1/x$, the height of the graph above that point, is whatever height the puddle must have to maintain an area of $1$.
+
+<Figure
+  image="figures/11.00.svg"
+  video="figures/11.11-11.24-trimmed.mp4" 
+/>
+
+For the derivative, imagine nudging the input $x$ up by a value $dx$. How must the height of this rectangle change so that the area remains unchanged at $1$? That is, increasing the width by $dx$ adds some new area to the right here, so the puddle must decrease in height by some $d(1/x)$ so that the area lost off the top here cancels that out.
+
+<Figure
+  image="figures/11.41.svg" 
+/>
+
+You should think of that $d(1/x)$ as being some tiny *negative* value, since it’s *decreasing* the height of this rectangle. And you know what? I’m gonna leave the last few steps here for you to pause and ponder. And once you work out $d(1/x)/dx$, compare it to what happens if you apply the power rule purely symbolically to $x^{-1}$.
+
+$$
+\frac{d(1 / x)}{d x}= ? ? ?
+$$
+
+
+## Comprehensive Questions
+
+Here are a couple questions to test your knowledge of derivatives and the past chapters.
+
+### Derivative of $f(x) = \sqrt{x}$
+
+Modeling the derivatives geometrically can be applied to many different types of functions. For example, see if you can reason your way through the derivative of $f(x) = \sqrt{x}$ which also can be written as $x^{\frac{1}{2}}$. Now, one way to find the derivative of this function is to recognize that it fits the mold of the power rule. Blindly applying the rule will produce an answer, but what does it *mean*?
+
+To start, try modeling function using geometry.
+
+<Figure
+  image="figures/12.34.svg"
+  caption="Unlike in previous problems, $dx$ does not represent geometric length, but instead represents **area**."
+/>
+
+<Question
+  question="What is the derivative of the function $f(x) = \sqrt{x}$?"
+  choice1="$\sqrt{2}x$"
+  choice2="$\frac{1}{2\sqrt{x}}$"
+  choice3="$\frac{x}{\sqrt{2}}$"
+  choice4="$\frac{1}{2}x$"
+  answer={2}
+>
+
+In order to find the derivative of the function $f(x) = \sqrt{x}$ we can model its geometry as the area of the square, where given the area $x$ the function returns the length of the square's side which is the square root of $x$.
+
+$$
+x = \sqrt{x} \sqrt{x} \\
+$$
+
+Then, we want to find $d\sqrt{x}$, the change of the function's output, when the input $x$ is nudged by a small value $dx$. Since $x$ represents area, it makes sense that a small nudge $dx$ should also be area! 
+
+<Figure
+  image="figures/12.34-explanation.svg"
+  caption="Unlike in previous problems, $dx$ is not geometric length, but instead represents **area**."
+/>
+
+The added area changes the output of the function by the new length $d\sqrt{x}$, which looks like this.
```

Comment:
So the question is, how much does the side length of that square increase if the total change to the area is meant to be $dx$?